### ISSUE TITLE
Le jeu m'a forcé à choisir 2 malédictions alors que j'avais fait mes 3 quêtes ...

### ISSUE THREAD

**Issue by Simon-Lessard** [JulienNigon/SpellcasterUniversity-Bug-Report#920]
Je ne sais pas si j'ai trigger une condition obscure qui m'a maudit ou si c'est un bug...

![image](https://user-images.githubusercontent.com/23580881/76459993-80ea9180-63dd-11ea-9d80-2f59d765d9bd.png)

**Comment by Hyslyne** [comment]
C'est un bug

**Comment by Neelan22** [comment]
Oui, c'est un bug... Est ce que cette malédiction en plus est arrivée sur la carte des volcans?
Tu as choisis 2 fois une malédiction ou la 2e est arrivée toute seule à coté de la première?

**Comment by Simon-Lessard** [comment]
En terminant le volcan puis passant au marais (qui fonctionne de nouveau).

J'ai choisi les deux malédictions, pas d'assignation automatique.

**Comment by Neelan22** [comment]
ok, merci

**Comment by Myu-Wynn** [comment]
Bonjour,

Je me permets de relancer car je viens de subir la même chose.
![20200516153511_1](https://user-images.githubusercontent.com/65197571/82121162-372a7880-978b-11ea-82ee-0f6480d20ce6.jpg)

Pour moi, c'est en voulant continuer ma campagne au Désert que le bug s'est produit (venant du Volcan).

**Comment by Neelan22** [comment]
C'était aussi après la carte du volcan? Je vois quelle a été joué, mais je ne peux pas voir si il s'agit de la dernière

**Comment by Myu-Wynn** [comment]
Oui, c'était après celle du volcan ! J'ai recommencé une campagne dans le
choisir en 3ème map, du coup :/
